pkg/controller/tiers/tiers_controller.go

```diff
@@ -55,10 +55,6 @@ var log = logf.Log.WithName("controller_tiers")
 // Add creates a new Tiers Controller and adds it to the Manager.
 // The Manager will set fields on the Controller and Start it when the Manager is Started.
 func Add(mgr manager.Manager, opts options.AddOptions) error {
-	if !opts.EnterpriseCRDExists {
```

Comment:
I think like I mentioned in slack I don't think we need to run this controller for Calico. We only need this if we're going to be deploying policies to the `allow-tigera` tier.
Unless allowing this to run makes everything else easier, I don't think we should run it with Calico.

Reply:
we create a policy in the allow-tigera tier to allow traffic from fluentd to guardian. we can create the allow-tigera tier via manifest before creating the log-collector CR?